frontend/.eslintrc.cjs

```diff
@@ -20,6 +20,7 @@ module.exports = {
   parser: "@typescript-eslint/parser",
   plugins: ["react-refresh", "jsx-a11y", "prettier", "@typescript-eslint"],
   rules: {
+    "jsx-a11y/no-autofocus": "off",
```

Comment:
We do want to manage focus in our application, so I suggest to disable this rule. Doing it with JS instead of autofocus has the same issues and is not a solution for this.